corehq/apps/reports/static/reports/js/hq_report.js

```diff
@@ -214,13 +216,13 @@ hqDefine("reports/js/hq_report", [
 
             self.send_to_owner = ko.observable(true);
             self.subject = ko.observable(hqReport.emailDefaultSubject);
-            self.recipient_emails = ko.observable();
+            self.recipient_emails = ko.observableArray();
```

Comment:
i have a feeling this not being an `observableArray` might have been the crux of the issue